src/zarr/abc/codec.py

```diff
@@ -11,18 +11,23 @@
 from zarr.buffer import Buffer, NDBuffer
 from zarr.chunk_grids import ChunkGrid
 from zarr.common import ChunkCoords, concurrent_map
-from zarr.config import config
+from zarr.config import Config, config
 
 if TYPE_CHECKING:
     from typing_extensions import Self
 
     from zarr.array_spec import ArraySpec
+    from zarr.common import JSON
     from zarr.indexing import SelectorTuple
 
 CodecInput = TypeVar("CodecInput", bound=NDBuffer | Buffer)
 CodecOutput = TypeVar("CodecOutput", bound=NDBuffer | Buffer)
 
 
+def get_config() -> Config:
+    return config
```

Comment:
at first glance, I don't understand what this function is doing. Do we need it? If so, would you mind adding a comment explaining what it is for?